Dockerfile

```diff
@@ -11,6 +11,7 @@ COPY --from=builder /hypershift/bin/hypershift \
                     /hypershift/bin/hcp \
                     /hypershift/bin/hypershift-operator \
                     /hypershift/bin/control-plane-operator \
+                    /hypershift/bin/control-plane-pki-operator \
```

Comment:
we also have `Containerfile.operator` and `fast.Dockerfile` that need to be updated.